### PR TITLE
Feature/create event

### PR DESCRIPTION
**Pull request by d151**
*Shortly describe the problem, feature or fix*

### Add
- 

### Change
- fixed all lint errors

### Remove
- 

### Changelog
- [ ] changelog updated with your changes

### Issuer
- https://github.com/lysnikolaou/eatandmeet/issues/TICKET_NUMBER

### CODE REVIEW

**Comment by nitzanashi** [comment]
Why did you closed the previous Pull Request? :(

**Comment by nitzanashi** [comment]
you ignored some of my previous comments and you can't see them here because you deleted the Previous pull request please recheck it , i requested some changes in the style, also change-log also update to master version... you just make everything harder when you close PR because you can't follow the requested changes

**Comment by d151** [comment]
bcause i couldnt open a new one i think. or maybe im a rookie and dont know how to use git that well

**Comment by nitzanashi** [comment]
you just needed to update the open one, just push more ocmmits...  the only change that you made was about the lint problem and not solved any of the other stuff like adding description for the form....

File: frontend/src/components/Navigation/index.jsx [hunk 1]
```diff
@@ -21,40 +21,84 @@ class Navigation extends Component {
     render () {
```
Comment (nitzanashi) [inline frontend/src/components/Navigation/index.jsx]: Update your branch to master, resolve conflicts and i will Aprove this Pull Request

**Pull request merged**